chalice/app.py

```diff
@@ -232,6 +232,11 @@ def get_access_control_headers(self):
 
         return headers
 
+    def __eq__(self, other):
```

Comment:
-0 for me.  Given this is just a value object, equality is pretty reasonable to me.  For a more complicated app it might make it easier to not have to track around the exact object you want to apply for a set of views.